test-network-nano-bash/README.md

```diff
@@ -0,0 +1,78 @@
+# Test network - Nano bash
+
+Test network Nano bash provides a set of minimal bash scripts to run a Fabric network on your local machine.
+The network is functionally equivalent to the docker-based Test Network, you can therefore run all the tutorials and samples that target the Test Network.
+The Fabric release binaries are utilized rather than using docker containers to avoid all unnecessary layers. Only the chaincode and chaincode builder runs in a docker container behind the scenes.
+Using the Fabric binaries also makes it simple for Fabric developers to iteratively and quickly modify Fabric code and test a Fabric network as a user.
+
+As the name `nano` implies, the scripts provide the smallest minimal setup possible for a Fabric network while still offering a multi-node TLS-enabled network:
+- Minimal set of dependencies
+- Minimal requirements on Fabric version (any v2.x orderer and peer nodes are supported)
+- Minimal set of environment variable overrides of the default orderer orderer.yaml and peer core.yaml configurations
+- Minimal scripting with minimal set of reference commands to get a Fabric network up and running
+- Minimal channel configuration for an orderer organization (3 ordering nodes) and two peer organizations (with two peers each)
+- Minimal endorsement policy to allow a single organization to approve and commit a chaincode
+
+# Prereqs
+
+- Follow the Fabric documentation for the [Prereqs](https://hyperledger-fabric.readthedocs.io/en/latest/prereqs.html)
+- Follow the Fabric documentation for [downloading the Fabric samples and binaries](https://hyperledger-fabric.readthedocs.io/en/latest/install.html). You can skip the docker image downloads by using `curl -sSL https://bit.ly/2ysbOFE | bash -s -- -d`
+
+# Instructions for starting network
+
+Open terminal windows for 3 ordering nodes, 4 peer nodes, and 4 peer admins as seen in the following terminal setup.
```

Comment:
Add in 2 orgs here?  Which org has which peer.

Reply:
Done.